### ISSUE TITLE
code in Query.cc seems to assume k==20

### ISSUE THREAD

**Issue by rsharris** [Kingsford-Group/splitsbt#5]
I was looking for something else but this caught my eye.

In splitQuery, Query.cc line 1476, the use of substr(0,19) might be assuming the kmer size is 20.  The comment above the loop says "Add k-1 nucleotides to ..." and it looks like that's exactly what it's doing if k is 20.

**Comment by Bradsol** [comment]
You are correct - however that's just a test function I wrote for a new syntax query. Although its admittedly hard to follow, splitQuery() is only used in splitQueryInfo [which is in turn used in batchInfo] none of which is actually being used in any main-accessible function. Basically I hard-coded some simple examples just to play around with an idea that never really worked the way I wanted it to.

The newer (unreleased) queries are exclusively using batchQuery and boolQuery objects while the old is unchanged. But let me know if you find any other issues!

**Comment by rsharris** [comment]
Understood.  I didn't look for how it was used, or even *if* it was used.
